packages/eslint-plugin/src/rules/prefer-function-type.ts

```diff
@@ -143,13 +143,12 @@ export default createRule({
                 isParentExported
               ) {
                 const commentsText = comments.reduce((text, comment) => {
-                  return (
+                  return `${
```

Comment:
This is one of those cases that I think benefits from having the interpolated expression be assigned a variable